templates/pioneer-study/esper-study-telemetry.1.schema.json

```diff
@@ -0,0 +1,225 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "defaultSearchEngine": {
+      "description": "",
+      "maxLength": 256,
+      "minLength": 1,
+      "type": "string"
+    },
+    "locale": {
+      "description": "",
+      "maxLength": 256,
+      "minLength": 1,
+      "type": "string"
+    },
+    "os": {
+      "description": "",
+      "maxLength": 256,
+      "minLength": 1,
+      "type": "string"
+    },
+    "normalizedChannel": {
```

Comment:
This is in the metadata, no need to send it with the ping/validate it.

Reply:
(Same as the other comment)